clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteredStoreConfiguration.java

```diff
@@ -25,13 +25,17 @@
  */
 public class ClusteredStoreConfiguration implements ServiceConfiguration<ClusteredStore.Provider> {
 
+  public static final Consistency DEFAULT_CONSISTENCY = Consistency.EVENTUAL;
+  public static final int DEFAULT_CONCURRENCY = 16;
+
   private final Consistency consistency;
+  private final int concurrency;
 
   /**
    * Creates a new configuration with consistency set to {@link Consistency#EVENTUAL EVENTUAL}.
    */
   public ClusteredStoreConfiguration() {
-    this(Consistency.EVENTUAL);
+    this(DEFAULT_CONSISTENCY, DEFAULT_CONCURRENCY);
```

Comment:
This change and all related ones are irrelevant for this PR and should not be part of it. Can they be removed?